styles/typography/typography-preset-1.json

```diff
@@ -118,7 +93,9 @@
 			},
 			"core/quote": {
 				"typography": {
-					"fontWeight": "300"
+					"fontSize": "var:preset|font-size|large",
+					"fontWeight": "300",
```

Comment:
This font weight and font size are is the same as in theme.json so it does not need to be duplicated.
The line height is the same as on the body so it does not need to be duplicated.